lang/python/python-mysql/Makefile

```diff
@@ -60,6 +66,8 @@ endef
 
 $(eval $(call PyPackage,python-mysql))
 $(eval $(call BuildPackage,python-mysql))
+$(eval $(call BuildPackage,python-mysql-src))
```

Comment:
Oh, forgot to mention this in the commit message -- not sure if that is an issue.